packages/router/src/adapters/subgraph/index.ts

```diff
@@ -37,7 +16,7 @@ export const getSdks = (): Record<number, Sdk> => {
   return sdks;
 };
 
-export const subgraphContractReader = (): ContractReader => {
+export const getSubgraphContractReader = async (): Promise<ContractReader> => {
```

Comment:
If we're renaming it why not rename it to `getContractReader`

Reply:
Because it's specifically a subgraph implementation of contract reader. The adapters should specify what type of real-world implementation it is.